fastlane-plugin-revenuecat_internal.gemspec

```diff
@@ -30,7 +30,9 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency('rspec_junit_formatter')
   spec.add_development_dependency('rubocop', '1.31.1')
   spec.add_development_dependency('rubocop-performance')
+  spec.add_development_dependency('rubocop-rake')
   spec.add_development_dependency('rubocop-require_tools')
+  spec.add_development_dependency('rubocop-rspec')
```

Comment:
Added a couple of extra rubocop dependencies since it was being suggested and didn't cause any harm.

Reply:
👍